### ISSUE TITLE
I use Arch, btw

### ISSUE THREAD

**Issue by adberns** [EnterpriseQualityCoding/FizzBuzzEnterpriseEdition#573]

**Comment by echoofskyz** [comment]
That's a lie!

**Comment by adberns** [comment]
But I've _seen_ people use Arch

**Comment by Rudxain** [comment]
Ridiculous! Nobody uses Arch on public spaces

**Comment by tori4582** [comment]
Hi @adberns

It seems like your machine is currently non-compliant with our Security Guidelines. It is strictly prohibited for non-Windows machines to work on this project due to highly-enforced security standards and intelligence proprietaries protection (especially from the recent Oracle legal strike which is mentioned in #652).

Please quickly contact your Tom's a truly friendly IT support to be issued a new machine with the latest version of Windows ME. This has to be done within 3 business days.

If you have any concerns, please do not hesitate to drop a message here. We will circle back very soon.

Best regards,

Steve

**Comment by GEOEGII555** [comment]
> It seems like your machine is currently non-compliant with our Security Guidelines. It is strictly prohibited for non-Windows machines to work on this project due to highly-enforced security standards and intelligence proprietaries protection (especially from the recent Oracle legal strike which is mentioned in #652).

It is also strictly prohibited for Windows machines that are **not** enrolled into our active directory company policies to work on this project. Make sure to enroll the computer into our Active Directory after installing the Microsoft(R) Windows(R) 11 24H2 secure operating system.

**Comment by Rudxain** [comment]
Don't forget to install the **Crowdstrike Security Solution**!

**Comment by GEOEGII555** [comment]
> Don't forget to install the **Crowdstrike Security Solution**!

Also don't forget to install the following security solutions:

1. Kaspersky Premium
2. Norton 360
3. Bitdefender Total Security
4. Avast Antivirus
5. Microsoft Defender
6. TotalAV
7. ClamAV
8. McAfee Security Solution

**Comment by Rudxain** [comment]
> Also don't forget to install the following security solutions [...]

Agreed. Security is of utmost importance here. If any ~slave~ employee complains that "My RAM is at 99%" just ask them "Do you want the company to be hacked?"

**Comment by GEOEGII555** [comment]
> > Also don't forget to install the following security solutions [...]
> 
> Agreed. Security is of utmost importance here. If any ~slave~ employee complains that "My RAM is at 99%" just ask them "Do you want the company to be hacked?"

(Or just install more RAM into the computers)

**Comment by Rudxain** [comment]
> (Or just install more RAM into the computers)

Budget too low, let's download RAM instead :fire:

**Comment by GEOEGII555** [comment]
> > (Or just install more RAM into the computers)
> 
> Budget too low, let's download RAM instead :fire:

Ask our Windows Active Directory system administrator to make a message pop up asking employees to open https://downloadmoreram.com/ since our Windows system administrator doesn't know how to write PowerShell scripts.
